.grype.yaml

```diff
@@ -3,3 +3,8 @@
 ignore:
   - vulnerability: CVE-2021-22570
   - vulnerability: CVE-2015-5237
+
+  # Bless your heart NPM, you're a mess
+  - vulnerability: CVE-2017-18589
```

Comment:
Can we link the github issues you found here? (so they're trackable)  We don't need them for all of them but would be good context to maintain

Reply:
There are 3 issues open for that one: https://github.com/anchore/grype/issues/931. And plenty more false positives I don't really think should break the build.
![Screen Shot 2022-09-22 at 2 54 21 PM](https://user-images.githubusercontent.com/882485/191838893-1009051d-c9d2-4136-921f-04b2da91ec19.png)

Reply:
Added https://github.com/defenseunicorns/zarf/issues/767